profile.json

```diff
@@ -1,11 +1,10 @@
 {
   "contact": {
-    "email": "anic1530@gmail.com",
-    "telegram": "@sungwoo8moon",
-    "twitter": "@8moonic",
-    "discord": "anic1530 | 8moon#5925"
+    "email": "info@luncitalia.com",
+    "telegram": "@LUNCCommunityitalia",
+    "twitter": "@LUNC_Italia"
   },
    "notifications": {
-       "email": "anic1530@gmail.com"
+       "email": "info@luncitalia.com"
   }
 }
```

Comment:
Hello!  Please update your `profile.json` in your validator directory:

https://github.com/terra-money/validator-profiles/blob/master/validators/terravaloper1ef4hd8agtx8unaut0fc99zs9928qzm6txjcx40/profile.json

Reply:
Ok, i have updated it

Reply:
> Ok, i have updated it

@LuncCommunityItalia Just delete the `profile.json` file outside of your validator directory and you should be good :)

<img width="314" alt="Screen Shot 2022-11-04 at 12 11 50 PM" src="https://user-images.githubusercontent.com/30275692/200024013-f950bb84-d4da-46dd-909b-0b4ffcfb306f.png">

Reply:
ok